docker/mysql/db/db_upgrade/orgchart/Update_RMC_DB_2023092100-2024071800.sql

```diff
@@ -0,0 +1,60 @@
+START TRANSACTION;
+
+ALTER TABLE `employee_data` DROP FOREIGN KEY `empUID_rel`;
+ALTER TABLE `employee_data` CHANGE `empUID` `empUID` int unsigned NOT NULL FIRST;
+ALTER TABLE `relation_position_employee` DROP FOREIGN KEY `empUID_rel_position`;
+ALTER TABLE `relation_position_employee` CHANGE `empUID` `empUID` int unsigned NOT NULL AFTER `positionID`;
+ALTER TABLE `relation_group_employee` DROP FOREIGN KEY `empUID_rel_group`;
+ALTER TABLE `relation_group_employee` CHANGE `empUID` `empUID` int unsigned NOT NULL AFTER `groupID`;
+ALTER TABLE `relation_employee_backup` DROP FOREIGN KEY `empUID_rel_backup`;
+ALTER TABLE `relation_employee_backup` DROP FOREIGN KEY `empUID_rel_backup2`;
+ALTER TABLE `relation_employee_backup` CHANGE `empUID` `empUID` int unsigned NOT NULL FIRST,
+    CHANGE `backupEmpUID` `backupEmpUID` int unsigned NOT NULL AFTER `empUID`;
+ALTER TABLE `employee_privileges` DROP FOREIGN KEY `empUID_rel_privs`;
+ALTER TABLE `employee_privileges` CHANGE `empUID` `empUID` int unsigned NOT NULL FIRST;
+ALTER TABLE `employee_data_history` DROP FOREIGN KEY `empUID_rel_history`;
+ALTER TABLE `employee_data_history` CHANGE `empUID` `empUID` int unsigned NOT NULL FIRST;
+ALTER TABLE `employee` CHANGE `empUID` `empUID` int unsigned NOT NULL AUTO_INCREMENT FIRST;
+
+ALTER TABLE `employee_data` ADD CONSTRAINT `empUID_rel` FOREIGN KEY (`empUID`) REFERENCES `employee`(`empUID`) ON DELETE CASCADE ON UPDATE RESTRICT;
+ALTER TABLE `relation_position_employee` ADD CONSTRAINT `empUID_rel_position` FOREIGN KEY (`empUID`) REFERENCES `employee`(`empUID`) ON DELETE CASCADE ON UPDATE RESTRICT;
+ALTER TABLE `relation_group_employee` ADD CONSTRAINT `empUID_rel_group` FOREIGN KEY (`empUID`) REFERENCES `employee`(`empUID`) ON DELETE CASCADE ON UPDATE RESTRICT;
+ALTER TABLE `relation_employee_backup` ADD CONSTRAINT `empUID_rel_backup` FOREIGN KEY (`empUID`) REFERENCES `employee`(`empUID`) ON DELETE CASCADE ON UPDATE RESTRICT;
+ALTER TABLE `relation_employee_backup` ADD CONSTRAINT `empUID_rel_backup2` FOREIGN KEY (`empUID`) REFERENCES `employee`(`empUID`) ON DELETE CASCADE ON UPDATE RESTRICT;

```

Comment:
```
...`empUID_rel_backup` FOREIGN KEY (`empUID`) REFERENCES `employee`(`empUID`)...
...`empUID_rel_backup2` FOREIGN KEY (`empUID`) REFERENCES `employee`(`empUID`)...
```
empUID_rel_backup2 is using the wrong foreign key. Same issue in the Revert DB section.

The correct foreign key is:
![image](https://github.com/user-attachments/assets/8a5e8c82-7e4b-426e-8ec0-fc15f9047d32)

Reply:
Change was made.